packages/medusa/src/commands/seed.js

```diff
@@ -26,15 +26,30 @@ const t = async function({ directory, migrate, seedFile }) {
       process.exit(1)
     }
   }
+  const configModule = await configLoader(directory)
+
+  let hostConfig = {
+    database: configModule.projectConfig.database_database,
+    url: configModule.projectConfig.database_url,
+  }
+
+  if (configModule.projectConfig.database_host) {
+    hostConfig = {
+      host: configModule.projectConfig.database_host,
+      port: configModule.projectConfig.database_port,
+      database: configModule.projectConfig.database_database,
+      ssl: configModule.projectConfig.database_ssl,
+      username: configModule.projectConfig.database_username,
+      password: configModule.projectConfig.database_password,
+    }
+  }
 
-  const { configModule } = getConfigFile(directory, `medusa-config`)
   const dbType = configModule.projectConfig.database_type
   if (migrate && dbType !== "sqlite") {
-    const migrationDirs = getMigrations(directory)
+    const migrationDirs = await Promise.resolve(getMigrations(directory))
```

Comment:
todo: await should suffice instead of Promise.resolve, because that utils become async

Reply:
done